commercetools-convenience/src/main/java/io/sphere/sdk/sequencegenerators/CustomObjectBigIntegerNumberGeneratorConfigBuilder.java

```diff
@@ -0,0 +1,75 @@
+package io.sphere.sdk.sequencegenerators;
+
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.customobjects.CustomObject;
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.models.Builder;
+
+import java.math.BigInteger;
+
+/**
+ *
+ * Builds {@link CustomObjectBigIntegerNumberGeneratorConfig} instances using the Builder pattern.
+ */
+public class CustomObjectBigIntegerNumberGeneratorConfigBuilder extends Base implements Builder<CustomObjectBigIntegerNumberGeneratorConfig> {
+
+    private SphereClient sphereClient;
+    private int maxRetryAttempts = 100;
+    private String container = "io.sphere.sdk.sequencegenerators";
+    private String key;
+    private BigInteger initialValue = BigInteger.ONE;
+
+    CustomObjectBigIntegerNumberGeneratorConfigBuilder(final SphereClient sphereClient, final String key) {
+        this.sphereClient = sphereClient;
+        this.key = key;
+    }
+
+    /**
+     *
```

Comment:
If you describe parameters you should also provide a description for the method otherwise people may not expect a parameter description in the Javadoc.